app/models/alchemy/page.rb

```diff
@@ -406,6 +406,16 @@ def update_node!(node)
       update_columns(hash)
     end
 
+    # Holds the +FixedAttributes+ class instance
```

Comment:
"Holds an instance of +FixedAttributes+" ("class instance" sounds strange, as if we're talking about the `Class` class of which `FixedAttributes`, the class, *is* an instance).